examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,146 @@
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.ExecAbortException;
+import glide.api.models.exceptions.TimeoutException;
+
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class StandaloneExample {
+
+    /**
+     * Creates and returns a <code>GlideClient</code> instance. This function initializes a
+     * <code>GlideClient</code> with the provided node address.
+     *
+     * @return A <code>GlideClient</code> connected to the provided node address.
+     * @throws CancellationException if the operation is cancelled.
+     * @throws ExecutionException if the client fails due to execution errors.
+     * @throws InterruptedException if the operation is interrupted.
+     */
+    public static GlideClient createClient() throws CancellationException, ExecutionException, InterruptedException {
+        String host = "localhost";
+        Integer port = 6379;
+
+        // Check `GlideClientConfiguration` for additional options.
+        GlideClientConfiguration config =
+                GlideClientConfiguration.builder()
+                        .address(NodeAddress.builder().host(host).port(port).build())
+                        // Enable this field if the servers are configured with TLS.
+                        //.useTLS(true);
+                        .build();
+
+        GlideClient client = GlideClient.createClient(config).get();
+        return client;
+    }
+
+    /**
+     * Executes the main logic of the application, performing basic operations such as SET, GET, and
+     * PING using the provided <code>GlideClient</code>.
+     *
+     * @param client An instance of <code>GlideClient</code>.
+     * @throws ExecutionException if an execution error occurs during operations.
+     * @throws InterruptedException if the operation is interrupted.
+     */
+    public static void appLogic(GlideClient client) throws ExecutionException, InterruptedException {
+
+        // Send SET and GET
+        CompletableFuture<String> setResponse = client.set("foo", "bar");
+        Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+
+        CompletableFuture<String> getResponse = client.get("foo");
+        Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+
+        // Send PING to the primary node
+        CompletableFuture<String> pong = client.ping();
+        Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+    }
+
+    /**
+     * Executes the application logic with exception handling.
+     *
+     * @throws ExecutionException if an execution error occurs during operations.
+     */
+    private static void execAppLogic() throws ExecutionException {
+
+        while (true) {
+            GlideClient client = null;
+            try {
+                client = createClient();
+                appLogic(client);
```

Comment:
return appLogic(client);

Reply:
Cannot return a value from a method with void result type. Can I propose the following?
```suggestion
                appLogic(client);
                return;
```